### PR TITLE
W4 - VietPhan - Load image

### PR DESCRIPTION
**Pull request by at-vietphan**
#### Related issues (Optional)
Task #

#### Description (Required)
- [x] Load image
- [x] Nine path

### CODE REVIEW

**Comment by at-vietphan** [comment]
@at-kietvo Please review this pull request. Thannks :+1:

File: app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/ImageVectorActivity.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import vn.asiantech.internship.R;
+
+/**
+ * Created by vietphan on 13/12/2017.
+ * Class ImageVectorActivity
+ */
+
```
Comment (at-kietvo) [inline app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/ImageVectorActivity.java]: Pls remove this line

File: app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/ImageVectorActivity.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+public class ImageVectorActivity extends AppCompatActivity {
+    @Override
```
Comment (at-kietvo) [inline app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/ImageVectorActivity.java]: Pls format code.

File: app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/LoadImageActivity.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+import java.io.File;
+
+import vn.asiantech.internship.R;
+
+/**
+ * Created by vietphan on 12/12/2017.
+ * LoadImageActivity: load image use picasso
+ */
+public class LoadImageActivity extends AppCompatActivity implements View.OnClickListener {
```
Comment (at-kietvo) [inline app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/LoadImageActivity.java]: Change class to `ImageLoaderActivity`

File: app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/LoadImageActivity.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                        .into(mImgPicasso);
+                break;
+            case R.id.btnAssets:
+                Picasso.with(this)
+                        .load("file:///android_asset/bg_desktop.png")
+                        .placeholder(R.mipmap.ic_launcher)
+                        .into(mImgPicasso);
+                break;
+            case R.id.btnStorage:
+                Uri uri = Uri.fromFile(new File(Environment.getExternalStorageDirectory().getPath() + "/Pictures/apple_mac_os_x_el_capitan.jpg"));
```
Comment (at-kietvo) [inline app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/LoadImageActivity.java]: Pls check file exist before load.

File: app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/NinePathActivity.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import android.support.v7.app.AppCompatActivity;
+
+import vn.asiantech.internship.R;
+
+/**
+ * Created by vietphan on 12/12/2017.
+ * NicePathActivity: load image nine path in edittext
+ */
+public class NinePathActivity extends AppCompatActivity {
+    @Override
```
Comment (at-kietvo) [inline app/src/at-vietphan/java/vn/asiantech/internship/ui/loadimage/NinePathActivity.java]: Reformat code.

File: app/src/at-vietphan/res/drawable/ic_image_vector.xml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="500px"
+    android:height="500px"
+    android:viewportHeight="500"
+    android:viewportWidth="500">
+    <path
```
Comment (at-kietvo) [inline app/src/at-vietphan/res/drawable/ic_image_vector.xml]: Reformat code.

File: app/src/at-vietphan/res/layout/activity_excercise.xml [hunk 7]
```diff
@@ -48,4 +48,9 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:text="@string/save_data" />
+    <Button
```
Comment (at-kietvo) [inline app/src/at-vietphan/res/layout/activity_excercise.xml]: Reformat code.

**Comment by at-vietphan** [comment]
I 'm fixed your comment. Please review this pull request. Thanks :+1:

**Pull request merged**